src/main/scala/diplomacy/BundleBridge.scala

```diff
@@ -0,0 +1,57 @@
+// See LICENSE.SiFive for license details.
+
+package freechips.rocketchip.diplomacy
+
+import chisel3._
+import chisel3.internal.sourceinfo.SourceInfo
+import freechips.rocketchip.config.{Parameters,Field}
+
+case class BundleBridgeParams[T <: Data](gen: () => T)
+case class BundleBridgeNull()
```

Comment:
could this be a `case object`?

Reply:
No other parameters classes are objects, so even if it's possible, I think it would be non-conforming.